moto/s3/responses.py

```diff
@@ -832,7 +832,10 @@ def _bucket_response_put(
                     bucket_name, self._acl_from_headers(request.headers)
                 )
 
-            if request.headers.get("x-amz-bucket-object-lock-enabled", "") == "True":
+            if request.headers.get("x-amz-bucket-object-lock-enabled", "") in [
```

Comment:
nit: The true strings could be put into a re-usable global constant like `TRUE_STRINGS = ["true", "True"]`. Perhaps such a constant already exists in the codebase @bblommers ?

Reply:
In other places we tend to solve this with `x.lower() == "true"`, so for consistency sake, would be good to have the same here